src/server/telemetry/TelemetryConstants.ts

```diff
@@ -0,0 +1,22 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+// Event Names
+export const AUTOCOMPLETE = 'autoComplete';
+
+// Properties - Keys
+export const SERVER = 'server';
+export const KEY_FOR_COMPLETION = 'keyForCompletion';
+export const TAG_FOR_COMPLETION = 'tagForCompletion'; // applicable only in htmlServer
+export const TIME_TAKEN_TO_PARSE = 'timeTakenToParse';
```

Comment:
prefer to use the 'Ms' suffix to make the units self-documenting. this also implies you are reporting in milliseconds.
Also, consider shortening to simply `'parseTimeMs'`